app/views/passengers/index.html.erb

```diff
@@ -0,0 +1,33 @@
+<strong>
+  <ul class="index_list">
+    <li>
+      ID
+    </li>
+    <li class="names">
+      Paasenger
+    </li>
+    <li>
+      Edit
+    </li>
+    <li>
+      Delete
+    </li>
+  </ul>
+</strong>
+
+<% @passengers.each do |passenger| %>
+<ul class="index_list">
```

Comment:
Watch your indentation in the views - when you utilize the code tags and HTML within, you still want to indent to make sure that it is as readable as possible.